src/logger.js

```diff
@@ -1,6 +1,5 @@
 import winston from 'winston'
 import c from './config'
-import {makeApiCall} from './slack/slackApi'
```

Comment:
the point of this task was also to get rid of `makeApiCall` function altogether, so please remove it

Reply:
removed `slackApi.js`